src/main/java/com/jfrog/ide/common/nodes/DependencyNode.java

```diff
@@ -149,4 +150,17 @@ public Object clone() {
     public String toString() {
         return getTitle();
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        DependencyNode that = (DependencyNode) o;
+        return indirect == that.indirect && Objects.equals(componentId, that.componentId) && Objects.equals(impactPaths, that.impactPaths) && Objects.equals(licenses, that.licenses) && Objects.equals(children, that.children);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(componentId, indirect, impactPaths, licenses);
```

Comment:
Does it intended that the `equals` implementation takes into account `Objects.equals(children, that.children);`, But the hashCode doesn't

Reply:
No, my mistake.